server/routes/bookings/bookingDetailsController.ts

```diff
@@ -43,8 +52,7 @@ export default class BookingDetailsController {
         visit,
         showCancelButton,
         showOLServiceNav: true,
-        isRequest: visit.visitSubStatus === 'REQUESTED',
-        isRejected: visit.visitSubStatus === 'AUTO_REJECTED' || visit.visitSubStatus === 'REJECTED',
+        isRequest: this.REQUEST_STATUSES.includes(visit.visitSubStatus),
```

Comment:
Simplifying here as the template doesn't need to know if it's a rejected request; just if it's a request is sufficient.